shared/api/event.go

```diff
@@ -18,3 +18,10 @@ type EventLogging struct {
 	Level   string            `yaml:"level" json:"level"`
 	Context map[string]string `yaml:"context" json:"context"`
 }
+
+// EventLifecycle represets a lifecycle type event entry
```

Comment:
Should add an API extension comment here (grep for API in shared/api for examples).